docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -0,0 +1,65 @@
+
+In the version vX.Y.Z of the AWS JDBC Driver, we have introduced a Telemetry feature to the wrapper that when enabled allows users to collect and visualize data of the execution of AWS JDBC Driver, at global level but also at plugin level, meaning that one can now inspect specific performance data for each plugin of the plugin chain.
+
+If you are using the AWS JDBC Driver, this enables you to monitor the performance of the driver with their configurations and determine whether that performance meets your expectation on the driver.
+
+## Terminology
+
+The AWS JDBC Driver feature provides telemetry data through two different forms: **Traces** and **Metrics**.
+
+### Traces
+
+Traces give an overview of what is happening in a specific section of the execution of an application. A trace is composed by a hierarchical sequence of segments, each of which contain basic information about the execution such as duration, and whether that section was executed with success or not.
+
+In the AWS JDBC Driver, initially a trace will be generated for every JDBC call made to the wrapper. And then, for each plugin that is part of the wrapper plugin chain, another segment will be created only for the plugin execution, linked to the JDBC call segment.
+
+Traces from the AWS JDBC Driver are submitted to **AWS X-Ray**. [link]
+
+### Metrics
+
+Metrics are numeric data that were measured and collected through the execution of an application. Those metrics can give an insight on how many times some action (such as failover) has happened and succeeded or failure, and others.
+
+The AWS JDBC Driver will submit metrics data to **AWS Cloudwatch**.
+
+The list of available metrics available for the AWS JDBC Driver and its plugins is available in the Appendix [link] section of this page.
+
+## Setting up the Amazon OTEL Collector
+
+## Pre Requirements
+
+Before enabling inside the application using the AWS JDBC Driver, a few setup steps are required to ensure the monitoring data gets properly emitted.
+
+1. In order to visualize the telemetry data in the AWS Console, make sure you have an IAM user or role with permissions to AWS X-Ray and AWS Cloudwatch.
+
+2. Download the AWS OTEL Collector and set it up. The AWS OTEL Collector is responsible from receiving telemetry data from the application using the AWS JDBC Driver and forward it to AWS. Both of those connections happen via HTTP, therefore URLs and ports need to be correctly configured for the collector.
+
+WARNING: The AWS OTEL Collector can be set up either locally either remotely. It is up to the user to decide where is best to set it up. If you decide to host it remotely, ensure that the application has the necessary permissions or whitelisting to connect to the Collector.
+
+WARNING: The collector is an external application that is not part of the wrapper itself. Without a collector, the wrapper will collect monitoring data from its execution but those will not be sent anywhere for visualization and get lost.
```

Comment:
Removing lost since the data is still collected, it just doesn't go anywhere. Lost may add confusion since the data is not being sent elsewhere.


```suggestion
WARNING: The collector is an external application that is not part of the wrapper itself. Without a collector, the wrapper will collect monitoring data from its execution but that data will not be sent anywhere for visualization.
```